knowledge-base/numerictextbox-disable-arrows.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Disabling NumericTextBox Arrows at Min or Max Value in Blazor
```

Comment:
People normally don't search in Google with gerund ("-ing"), so it's better that we don't use this grammar form in titles.